test/specs/integration.spec.js

```diff
@@ -50,6 +50,7 @@ describe('Integration tests', function() {
   beforeEach(function(done) {
     // Make sure we're connected to Firebase.  This can take a while on slow
     // connections.
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 180000;
```

Comment:
@mikelehen I was finally able to get the tests to pass on travis by passing only added timeout only to tests that were timeing out. I agree with not wanting to make it wait for too long on a failure, so I made sure to only put the timeouts where necessary (it seems like the `beforeEach` needed it more than anything else)

Totally open to not modifying timeout amounts, but this is literally the only way I was able to get it to pass on travis.